### DISCUSSION TITLE
Video 142. Turning custom datasets into DataLoaders: "RuntimeError: DataLoader worker"

### DISCUSSION THREAD

**Discussion by dsnal** [mrdbourke/pytorch-deep-learning#107] [General]
Time Stamp [22:25:56](https://youtu.be/Z_ikDlimN6A?t=80756)

If you are running this video on your local machine, you might run into a problem like this:

```
"RuntimeError: DataLoader worker (pid(s) 39691, 39693, 39695) exited unexpectedly" 
```

You can fix this problem by removing "`num_workers`"  from both `train_dataloader_custom` & `test_dataloader_custom` to disable parallel load.

**Comment by mrdbourke** [comment]
Thank you for tip @dsnal! 

I've added this note to the collection of Learn PyTorch in a day YouTube video fixes: https://github.com/mrdbourke/pytorch-deep-learning/discussions/98
